convert_proto.sh

```diff
@@ -0,0 +1,6 @@
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/..
+
+# NEEDS TO BE SET MANUALLY
+ENV_DIR=$CONDA_PREFIX
+
+$ENV_DIR/bin/2to3 --output-dir=$SCRIPT_DIR/pybmix/proto -W -n $SCRIPT_DIR/pybmix/proto
```

Comment:
This is kind of bad practice: we cannot assume that all the users will base their environment on conda.
It is better to ask directly to have 2to3 in the path